libr/core/gdiff.c

```diff
@@ -159,3 +159,79 @@ R_API void r_core_diff_show(RCore *c, RCore *c2) {
 		}
 	}
 }
+
+/* Iterate available diffs and print json output */
+R_API void r_core_diff_show_json(RCore *c, RCore *c2) {
+	RList *fcns = r_anal_get_fcns (c->anal);
+	const char *match;
+	RListIter *iter;
+	RAnalFunction *f;
+	PJ *pj = pj_new ();
+
+	if (!pj) {
+		return;
+	}
+
+	fcns = r_anal_get_fcns (c->anal);
+	if (r_list_empty (fcns)) {
+		eprintf ("No functions found, try running with -A or load a project\n");
```

Comment:
Use the new R_LOG_INFO() macro

Reply:
Does it make sense to use `R_LOG_ERROR()` here?